lib/services/events/remote_events.php

```diff
@@ -44,10 +44,18 @@ public function getLocalEvents() {
   /**
    * Performs the request for the events
    */
-  private function makeRequest($params = array()) {
+  private function makeRequest($params = array(), $count_only = false) {
     $requestUri = self::FEED_ENDPOINT;
     $requestUri .= '?' . http_build_query($params);
-    $response = file_get_contents($requestUri);
+    if($count_only){
```

Comment:
For the build to pass we just want to throw some spaces inbetween
```php
if ($count_only) {
```

https://travis-ci.org/SandersForPresident/SitesForBernie/jobs/82676125#L550